lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingLink.cs

```diff
@@ -84,8 +81,8 @@ public WritableLink(IPEndPoint remoteEndpoint, IInjector injector)
         /// Uses the already connected TcpClient.
         /// </summary>
         /// <param name="client">The already connected client</param>
-        /// <param name="injector">The injector to pass arguments to incoming messages</param>
-        public WritableLink(TcpClient client, IInjector injector)
+        /// <param name="streamingCodec">Streaming codec</param>
+        public StreamingLink(TcpClient client, IStreamingCodec<T> streamingCodec)
```

Comment:
These two constructors contain much duplicate code. It would be better to have both of them bottom out in the canonical constructor that gets all fields passed in. That constructor can be `private`, as it won't be used from the outside.